ubuntu_install_requirements.sh

```diff
@@ -47,6 +48,7 @@ if [[ $PATH != *pyenv* ]]; then
 fi
 
 pyenv install -s 2.7.5
+pyenv install -s 3.4.0
```

Comment:
I'm not familiar with python3 status on Ubuntu, but does 3.4 support align with a particular release?

For CentOS/RHEL we use python2 on CentOS/RHEL7 and python3 on CentOS/RHEL8 so I'm wondering if we could do something similar for Ubuntu - mixing python versions could get confusing/fragile otherwise perhaps?

Reply:
For now, I am working with upgrading the whole dev-env to python3. I just checked that eventually python latest is now 3.8. I will check and update the code again.